data_dictionary/Dockerfile

```diff
@@ -5,6 +5,6 @@ RUN chown -R nginx:nginx /var/cache/nginx
 RUN chown -R nginx:nginx /var/log/nginx
 RUN touch /var/run/nginx.pid && chown -R nginx:nginx /var/run/nginx.pid
 
-RUN apk -U upgrade
+RUN apt update --nocache
```

Comment:
I think `apt update` only updates the packages lists, you need to do a `apt upgrade` to install the new packages

Reply:
done